bookkeeper-server/src/main/java/org/apache/bookkeeper/auth/BookieAuthProvider.java

```diff
@@ -79,16 +75,19 @@ default void close() {}
     }
 
     /**
+     * Callback to let the provider know that the underlying protocol is changed.
```

Comment:
I think this is a part of a previous commit, that has already been merged on master.
The idea is that the AuthProvider will have access to the Certificates, in order to provide custom banning/revoking and automatic close of connection after certificate expiration